app/templates/dataset.html

```diff
@@ -223,9 +230,27 @@ <h2>Dataset README information</h2>
 <script type="text/javascript">
 
   function downloadVersion(event) {
+
+    let downloadButton = document.getElementById("downloadButton");
+    downloadButton.textContent = "Loading...";
+
+    let downloadButtonSpinner = document.getElementById("downloadButtonSpinner");
+    downloadButtonSpinner.hidden = false
+
+    let downloadButtonError = document.getElementById("downloadButtonError");
+    downloadButtonError.hidden = true
+
     fetch(`${window.origin}/download_content?id={{ data.id }}&version={{ metadata.get('version', 'latest') }}`)
       .then((response) => {
-        return response.blob();
+        console.log(response);
```

Comment:
@liamocn Could you remove that console.log please?

Reply:
@xlecours done!